platform/engineering/design-docs/2020-08-11-redirects-from-cms.md

```diff
@@ -0,0 +1,224 @@
+# Front End Build Redirects from CMS
+
+_Replace the previous line with the the title of your project or component and replace the following lines with your name(s), email and the date._
+**Author(s):** Steve Wirt
+**Last Updated:** Aug 21, 2020
+**Status:** Draft | **In Review** | Approved
+**Approvers:**
+   - [ ] Rian Fowler
+   - [ ] Andrew Gunsch
+   - [ ] Mikki Northuis
+   - [ ] Nick Sullivan
+   - [ ] Taylor Skinner (originally worked on redirect import)
+   - [ ] ??
+
+## Overview
+
+
+### Objective
+The CMS allows editors to create pages and move pages to other locations within the VA.gov website.  The CMS keeps track of when a page's path is changed and builds a redirect from one to the other.  The problem is, there is nothing that moves the redirects known within the CMS to the public website. It is in the best interest of editors, developers and veterans to have the front-end website know about and act on the redirects known to the CMS.
+
+### Background
+If pages are moved within the CMS under the right conditions the end result is a combination of internal broken links that currently cause failure of CMS content to be released to the front-end, and 404 errors for veterans.
```

Comment:
what are "the right conditions" that cause this? is this the table you have below?

Reply:
Yes that is covered by the conditions in the table.